src/components/report-filter-controls.tsx

```diff
@@ -106,6 +119,37 @@ function ReportFilterControls({ controls }: ReportFilterControlsProps): VNode {
                   </div>
                 </div>
               </fieldset>
+              {controls?.includes(Control.TIME_BUCKET) && (
+                <fieldset className="usa-fieldset">
+                  <legend className="usa-legend">Time Bucket</legend>
+                  <div className="usa-radio">
+                    <input
+                      type="radio"
+                      id="time-bucket-day"
+                      name="timeBucket"
+                      value="day"
+                      checked={timeBucket === TimeBucket.DAY}
+                      className="usa-radio__input"
+                    />
+                    <label htmlFor="time-bucket-day" className="usa-label usa-radio__label">
+                      Day
+                    </label>
+                  </div>
```

Comment:
Future refactor idea: make a component for these radio controls, it's a lot of markup and manual ID construction for only 2 options